"Cannot find module: 'tslib'"

I'm trying to use this in my Vite + TypeScript + React project. I'm getting this error:

> Error: Cannot find module 'tslib'
Require stack:
> - /Users/work/ProgrammingProjects/focusqueue/node_modules/.pnpm/@bradgarropy+use-countdown@1.5.2_react@18.2.0/node_modules/@bradgarropy/use-countdown/dist/index.js
 ❯ Object.<anonymous> node_modules/.pnpm/@bradgarropy+use-countdown@1.5.2_react@18.2.0/node_modules/@bradgarropy/use-countdown/dist/index.js:3:15

I found the reference to `tslib` in the compiled dist:

```
"use strict";
exports.__esModule = true;
var tslib_1 = require("tslib");
var countdown_1 = tslib_1.__importDefault(require("./countdown"));
exports["default"] = countdown_1["default"];
tslib_1.__exportStar(require("./types"), exports);
//# sourceMappingURL=index.js.map
```

Looking in `package.json` in this repo, it appears that `tslib` is a dev dependency.

Should that be moved to the regular dependencies list, or should the build step be modified somehow to remove the reference to `tslib` from the compiled distribution?

Can we re-open this?
The reason this error occurs with vite [is formulated here](https://github.com/angular/angular-cli/issues/13886#issuecomment-471927518).

Reproduce:
```shell
npm create vite@latest tslib-bug -- --template react-swc-ts
cd tslib-bug
npm install
npm add @bradgarropy/use-countdown
# edit src/App.tsx to import and use use-countdown
npm dev
```

Result:
  ➜  press h to show help
✘ [ERROR] Could not resolve "tslib"

    node_modules/@bradgarropy/use-countdown/dist/index.js:3:22:
      3 │ var tslib_1 = require("tslib");
        ╵                       ~~~~~~~

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "@bradgarropy/use-countdown",
-    "version": "2.0.0",
+    "version": "2.0.1",
     "description": "⏳ useCountdown hook",
     "keywords": [
         "javascript",

--- a/package.json
+++ b/package.json
@@ -43,7 +43,8 @@
         "prepare": "tsc"
     },
     "dependencies": {
-        "date-fns": "^2.19.0"
+        "date-fns": "^2.19.0",
+        "tslib": "^2.6.2"
     },
     "devDependencies": {
         "@bradgarropy/eslint-config": "^1.1.0",
@@ -72,7 +73,6 @@
         "react": "^17.0.1",
         "react-dom": "^17.0.1",
         "ts-jest": "^29.0.3",
-        "tslib": "^2.1.0",
         "typescript": "^4.1.3"
     },
     "peerDependencies": {

